.github/workflows/test-server-all.yml

```diff
@@ -32,7 +33,6 @@ jobs:
 
   small_test_cc:
     runs-on: ubuntu-latest
-    needs: [lint]
```

Comment:
What was the reason for removing `lint`?

Reply:
There is no reason to leave `lint` at "need".
The end time of the test will be delayed by about 30 seconds if we leave it as it is.

Reply:
> The end time of the test will be delayed by about 30 seconds if we leave it as it is.

wow... I understood. Thank you!